core/src/test/java/com/google/instrumentation/stats/StatsContextFactoryTest.java

```diff
@@ -24,8 +24,14 @@
  */
 @RunWith(JUnit4.class)
 public class StatsContextFactoryTest {
-  @Test(expected = IOException.class)
+  @Test
```

Comment:
Yes, I talked with Alistair yesterday and we think since the encoding has changed to [tag_metadata key_len key_bytes value_len value_bytes]*, an empty byte array will indicate that user doesn't specify any tag and uses the default tags. Therefore this case is allowed now.

While in the past the encoding format is num_tags [key_len key_bytes value_len value_bytes]*, even if a user doesn't specify a tag, there will be one type indicating the number of tags (0 in this case), and the input byte array should never be empty.